### PR TITLE
Update supported helm versions

### PR DESCRIPTION
**Pull request by bambash**
Updated tests and README to include supported versions of Helm.

### CODE REVIEW

File: end_to_end_testing/setup_helm3.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 source "$( cd -P "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"/setup_common.sh
 
 echo "Installing Helm"
```
Comment (ejether) [inline end_to_end_testing/setup_helm3.sh:3]:
```suggestion
echo "Installing Helm 3.1"
```

**Comment by ejether** [comment]
Is there something special about 2.12 that it needs it's own tests or can `helm 2` in this context mean that lates version of `helm 2`?

**Comment by bambash** [comment]
I figured if we're calling out a minimum version to support we should include it in end to end testing. I'm fine removing the extra test if we don't find value in it.

**Comment by ejether** [comment]
That is good enough logic for me. I'm good with it.

**Pull request merged**